web/ui/react-app/src/Navbar.tsx

```diff
@@ -37,12 +37,12 @@ const Navigation: FC<NavbarProps> = ({ consolesLink }) => {
             </NavItem>
           )}
           <NavItem>
-            <NavLink tag={Link} to={`${pathPrefix}/alerts`}>
+            <NavLink tag={Link} to="/alerts">
```

Comment:
is it safe to remove `pathPrefix` here ?

Reply:
Yeah, I noticed that in React Router, the `<Link>` tags already are relative to the base path.